tests/e2e/array-info.sh

```diff
@@ -39,7 +39,8 @@ export PFLEX_TENANT_PREFIX="tn1"
 export PSCALE_CLUSTER="Isilon-System-Name"
 export PSCALE_USER="username"
 export PSCALE_PASS="password"
-export PSCALE_ENDPOINT="1.1.1.1:8080"
+export PSCALE_ENDPOINT="1.1.1.1"
```

Comment:
Why is this change required for this version of the csm-operator? Why wasnit it required for 1.6.0?

Reply:
Technically speaking, this change is not required for it to work, because the field is replaced with whatever the end-user actually wants their array credentials to be: it's just a placeholder value. That placeholder value SHOULD only be an IP.

However, leaving it as 1.1.1.1:8080 communicates the wrong information to the user, that they should replace the value as `<IP>:<PORT>` when they really should just replace it as `<IP>`. Leaving it as it was could lead someone to put `<IP>:<PORT>` when running it themselves, and they will meet issues where it tries to deploy to `<IP>:<PORT>:<PORT>` which won't work. Removing the port from this line resolves that confusion/problem.